.travis.yml

```diff
@@ -15,6 +15,8 @@ install:
   # dependencies via pip.
   - conda create --yes -n test-env -c biocore python=$PYTHON_VERSION scikit-bio jupyter click nose scikit-learn
   - source activate test-env
+  - conda install --yes -c bioconda blast=2.6.0
```

Comment:
Thanks for asking! Yes, this is ``blast+``, though the package name is not clear. See the package info [here](https://anaconda.org/bioconda/blast). version 2.6.0 is the latest version, released in late January 2017.